index-scheduler/src/lib.rs

```diff
@@ -662,7 +662,11 @@ impl IndexScheduler {
         let rtxn = self.env.read_txn()?;
         self.index_mapper.index(&rtxn, name)
     }
-
+    /// Return the boolean referring if index exists.
+    pub fn index_exists(&self, name: &str) -> Result<bool> {
+        let rtxn = self.env.read_txn()?;
+        self.index_mapper.index_exists(&rtxn, name)
+    }
```

Comment:
Ahah! I didn't remember we had this method 🤩

```suggestion
    pub fn index_exists(&self, name: &str) -> Result<bool> {
        let rtxn = self.env.read_txn()?;
        self.index_mapper.index_exists(&rtxn, name)
    }

```